Sources/Perform/UITabBarController.swift

```diff
@@ -24,7 +24,7 @@ extension UITabBarController {
   ///     A function that will be invoked once with each view controller
   ///     in each tab's view controller hierarchy.
   public func configureViewControllers(_ configure: (UIViewController) -> Void) {
-    let hierarchy = (viewControllers ?? []).lazy.flatMap { $0.hierarchy }
+    let hierarchy: [UIViewController] = (viewControllers ?? []).lazy.flatMap { $0.hierarchy }
```

Comment:
Was this a compile error? IIRC I was intending for this type to be inferred as lazy so that the sequence wouldn't be consumed until the `forEach()` call below.

Reply:
It was. Specifying the type shouldn't affect its being lazy.